src/plugin/_locales/en/messages.json

```diff
@@ -189,7 +192,7 @@
     "message": "Something went wrong during the registration. Try again"
   },
   "register_ok": {
-    "message": "User succesfully registered. Now you can log in"
+    "message": "User succesfully registered. You will receive an email with instruccions to validate your account. You need to validate the account prior to log in."

```

Comment:
instructions not instruccions

Reply:
thx! it's a catalan typo.. ("instruccions" is catalan word for "instructions") :P

Reply:
fixed! now I don't know why it says that I dismissed your review.. sorry, was not my intention when I did the typo fix commit..

Reply:
no worries.!